R/loadComp.R

```diff
@@ -208,7 +208,9 @@ loadComp <- function(reg.model,
 predictSolute.loadComp <- function(
   load.model, flux.or.conc, newdata, interval=c("none","confidence","prediction"), 
   level=0.95, lin.or.log=c("linear","log"), se.fit=FALSE, se.pred=FALSE, date=FALSE, 
-  attach.units=FALSE, fit.reg=FALSE, fit.resid=FALSE, fit.resid.raw=FALSE, ...) {
+  attach.units=FALSE, fit.reg=FALSE, fit.resid=FALSE, fit.resid.raw=FALSE, 
+  agg.by=c("unit", "day", "month", "water year", "calendar year", "total", 
```

Comment:
I think that S3 consistency warning might be as simple to fix as adding `agg.by` after `attach.units` rather than after `fit.resid.raw`, but check out https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Generic-functions-and-methods as well.

(the warning: https://github.com/USGS-R/loadflex/pull/219#issuecomment-323098246)